packages/material-ui/src/Drawer/Drawer.js

```diff
@@ -49,18 +49,10 @@ export const styles = theme => ({
     left: 0,
     right: 'auto',
   },
-  paperAnchorLeftRounded: {
```

Comment:
I created this to give to everyone the option to use drawer/sidebars that should be square and use the same component with just square = false, to use sheets like the images below

![image](https://user-images.githubusercontent.com/11433064/41810210-11a19dfa-76d1-11e8-8fba-a4ca2ef7f613.png)

![image](https://user-images.githubusercontent.com/11433064/41810228-84733b90-76d1-11e8-8eae-e34b7ddbcd4e.png)

Reply:
@itelo Can we work on it in a different pull-request? Do you have a reference in the specification? I can't find it in https://material.io/design/components/sheets-side.html.

Reply:
Yes, and we could also add it in docs. I didn't find either.

Reply:
@itelo I think `shape` is not included in the docs yet. 

Also, how can I set border-radius on different window sizes? Thanks.